lib/resolve-repositories.js

```diff
@@ -8,7 +8,7 @@ export async function resolveRepositories(state, repositories) {
   }
 
   const repositoriesWithStars = repositories.filter((fullName) => {
-    return /^[a-z0-9_.-]+\/\*$/i.test(fullName);
+    return /^[a-z0-9_.-]+\/[a-z0-9_.-]*\*$/i.test(fullName);
```

Comment:
this will only match stars at the end, not in the middle or beginning, how about this:

```suggestion
    return /^[a-z0-9_.-]+\/[^*]*\*[^*]*$/i.test(fullName);
```

This will however only match patterns with a single `*`, e.g. not `octoherd/script-*-something-*`. But Maybe that is good enough, no need to make it too complicated. We could change the regex to support multiple stars:

```suggestion
    return /^[a-z0-9_.-]+\/[^*]*(\*[^*]*)*$/i.test(fullName);
```

But just because we can doesn't mean we should. 

Eventually we should probably rely on a dedicated pattern matching library, I'd suggest to keep it simple until the need for more complexity arises. Let me know what you think

Reply:
Agree on not overcomplicating it.

I will come up (tomorrow in my timezone) with a collection of potential input/outputs and maybe we can TDD them under a new issue and use that issue to introduce the library you are referencing and *for the moment, to just support the pattern `starts with`*

This way we keep landing small pieces of code at a time :)